src/sonic-py-common/sonic_py_common/syslogger.py

```diff
@@ -35,6 +39,74 @@ def __init__(self, log_identifier=None, log_facility=DEFAULT_LOG_FACILITY, log_l
         self.logger.addHandler(handler)
 
         self.set_min_log_priority(log_level)
+        
+        if enable_runtime_config:
+            self.update_log_level()
+                            
+    def update_log_level(self):
+        """Refresh log level.
+
+        Returns:
+            tuple: (refresh result, fail reason)
+        """        
+        from swsscommon import swsscommon
+        try:
+            config_db = swsscommon.SonicV2Connector(use_unix_socket_path=True)
+            config_db.connect(CONFIG_DB)
+            log_level_in_db = config_db.get(CONFIG_DB, f'{swsscommon.CFG_LOGGER_TABLE_NAME}|{self.log_identifier}', FIELD_LOG_LEVEL)
+            if log_level_in_db:
+                self.set_min_log_priority(self.log_priority_from_str(log_level_in_db))
+            else:
+                data = {
+                    FIELD_LOG_LEVEL: self.log_priority_to_str(self._min_log_level),
+                    FIELD_REQUIRE_REFRESH: 'true'
```

Comment:
@Junchao-Mellanox where is this refresh used?

Reply:
It is used by CLI. 

- If refresh flag is true, CLI will send SIGHUP to the daemon to refresh the log level. This is for Python logger
- If refresh flag is false, CLI will not send SIGHUP to the daemon, and daemon should have a thread to handle the log level update. This is for C/CPP logger which already has an extra thread to handle log level change.